source/page/Main.tsx

```diff
@@ -92,7 +94,7 @@ export class MainPage extends mixin() {
               >
                 ECMA-{standard}
               </Badge>
-              {standard === '262' ? (
+              {standard === '262' && edition <= thisYear ? (
```

Comment:
2020 年的规范是 2021 年发布的，因此是 
```suggestion
              {standard === '262' && edition < thisYear ? (
```

Reply:
![image](https://user-images.githubusercontent.com/19969570/94339326-c6924300-002b-11eb-897a-d17701e16644.png)
你这么改的话，这个有效链接也显示不出来了

Reply:
那么应该是在今年以前的有链接行为
在此之后的没有

Reply:
所以 `<=` 有何问题？

Reply:
好像没问题。。